src/Confluent.Kafka/Admin/AlterConfigsException.cs

```diff
@@ -34,12 +34,12 @@ public class AlterConfigsException : Exception
         ///     (whether or not they were in error). At least one of these
         ///     results will be in error.
         /// </param>
-        public AlterConfigsException(List<AlterConfigsExceptionResult> results)
-            : base(
+        public AlterConfigsException(List<AlterConfigsReport> results)
```

Comment:
`AlterConfigsReport` is `AlterConfigsResult`, consistent with `DeliveryReport` vs `DelveryResult`. nice :-). `..ExceptionResult` was an annoyingly horrible name.